### PR TITLE
Updated so now we compose a hana::map instead of inherit from it.

### PR DESCRIPTION
**Pull request by jlconlin**

### CODE REVIEW

File: src/constants/CODATA2018/test/CODATA2018.test.cpp [hunk 1]
```diff
@@ -83,5 +83,5 @@ SCENARIO("test all the constants"){
     )
   );
 
-  checkMap( aliasMap );
+  // checkMap( aliasMap );
```
Comment (whaeck) [inline src/constants/CODATA2018/test/CODATA2018.test.cpp]: why make a map and not test it?

**Review by whaeck — CHANGES_REQUESTED** [review]

File: src/constants/CODATA2018/test/CODATA2018.test.cpp (hunk 1 above)
Comment (jlconlin) [reply src/constants/CODATA2018/test/CODATA2018.test.cpp]: Good catch! This was removed while things were not working well. It is now back in.

**Comment by jlconlin** [comment]
This is ready to go whenever someone wants to approve it.

**Review by whaeck — APPROVED** [review]
I did not look at the actual numbers and units, since I assume they were already verified.

**Pull request merged**